testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -5421,6 +5421,47 @@ QUnit.module("regression", {
         assert.equal($.trim($tagContainer.text()), "Item1Item3", "selected values are rendered");
     });
 
+    QUnit.test("selection should work with pregrouped data without paging and with preloaded datasource", (assert) => {
+        const ds = new DataSource({
+            store: [
+                { key: "Category 1", items: [{ id: 11, name: "Item 11" }, { id: 12, name: "Item 12" }] },
+                { key: "Category 2", items: [{ id: 21, name: "Item 21" }, { id: 22, name: "Item 22" }] }
+            ]
+        });
+        ds.load();
```

Comment:
So we can remove it?
```suggestion

```

Reply:
No, i have created a new test without load. The TagBox should work both with preloading and without it